JsonPatch/Extensions.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+using Json.More;
+using Json.Pointer;
+
+namespace Json.Patch
+{
+	public static class Extensions
+	{
+		/// <summary>
+		/// Create list of PatchOperation from difference between original and modified
+		/// </summary>
+		/// <param name="originalObject">Original object</param>
+		/// <param name="modifiedObject">Modified object</param>
+		/// <param name="options">Json serializer options</param>
+		/// <typeparam name="T">Any object</typeparam>
+		/// <returns>JsonPatch</returns>
+		public static JsonPatch CreatePatch<T>(this T originalObject, T modifiedObject, JsonSerializerOptions options = null)
```

Comment:
I don't think that having the serializer options is necessary here.  Any conversions to/from JSON are completely internal.  You give an object (two really) and you get a `JsonPatch` out.

Reply:
Options are really necessary. Let me explain why.
By default serializer will generate sort of not correct json because of PascalCase naming, but asp.net generate json in camelCase by default, so web API in most cases is camelCase. So when you will got external patch you will got paths in camelCase, but serializing objects by default will have not same naming, so patch won't work. So as option there should be options here.

Reply:
BTW without options Add or Remove Test would be impossible in object context, because serializer always include all null values in JSON, so as you can see I ignore them  in options to make test possible. Yes I could do this via JsonDocument manually but it's C# and we are operating objects most of time, right?

Reply:
Yeah, I was thinking that the input -> output was always a .Net object, but the output is actually the patch.  I can see the need for the options now.